internal/pkg/service/buffer/api/gen/buffer/service.go

```diff
@@ -133,7 +133,7 @@ type CreateReceiverPayload struct {
 	ID *ReceiverID
 	// Human readable name of the receiver.
 	Name string
-	// Description of teh receiver.
+	// Description of the receiver.
```

Comment:
Minule som tu fixol aj toto typo, ale zabudol som pregenerovat kod, tak je to v tomto PR.